server/src/test/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1TestBase.java

```diff
@@ -26,12 +26,20 @@
 import com.linecorp.armeria.common.HttpHeaders;
 import com.linecorp.armeria.common.HttpMethod;
 import com.linecorp.armeria.common.MediaType;
+import com.linecorp.centraldogma.server.CentralDogmaBuilder;
 import com.linecorp.centraldogma.testing.CentralDogmaRule;
 
 public class ContentServiceV1TestBase {
 
     @Rule
-    public final CentralDogmaRule dogma = new CentralDogmaRule();
+    public final CentralDogmaRule dogma = new CentralDogmaRule() {
+        @Override
+        protected void configure(CentralDogmaBuilder builder) {
+            // Shorten the default request timeout here, in order to do a test that a watch request overrides
+            // the default request timeout.
```

Comment:
How about mentioning `watchRepositoryTimeout()` and `watchFileTimeout()` so people knows where to look?